utils/datetime.ts

```diff
@@ -24,9 +24,7 @@ export function isWithin(
     truncateEndpoints = shouldTruncate.get(unit) || false,
 ): boolean {
     const diff = getDiff(a, b, timeZone, unit, truncateEndpoints);
-    return threshold >= 0 ?
-        diff <= threshold && diff >= 0 :
-        diff >= threshold && diff <= 0;
+    return threshold >= 0 ? diff <= threshold && diff >= 0 : diff >= threshold && diff <= 0;
```

Comment:
0/5 I always felt the ternary operator too confusing. Here, where the "inside" of each part is kind of complex, even more. I liked better the multiline approach, even if it was a bit "strange". I would even like better the "golang way". Instead of `return x ? a : b` I would go with:
```
if (x) {
  return a
}
return b
```

Reply:
Similar comments with other uses of the ternary operator.

Reply:
this is basically a question of personal preference. IMO  this is just as readable as the other approach, but I am well used to this pattern, so it might not be the case for everybody. afaik the ternary does not have any impact on performance, since it is only syntactic sugar.

I am fine with writing it the "golang way".

@hmhealey Thoughts?